test/deprecated/vectorizeOnly.good

```diff
@@ -0,0 +1,4 @@
+vectorizeOnly.chpl:1: warning: 'vectorizeOnly()' is deprecated in favor of using 'foreach' loops
```

Comment:
It feels slightly confusing to me that 3 warnings are generated for one call, here. However I'm not so sure there's anything we can do about it.

Reply:
Yeah, I just (re?) noticed that this afternoon as well, and was going to look into it a bit more.  I'm guessing it's something that the compiler is doing that may be hard to un-do, but I'll check.

Reply:
I haven't been able to (easily) figure out what's going on.  Looking at the IR in the log files, it looks like there's only one call and that it's only getting invoked once.  Also, if I put a similar deprecation tag on a user iterator, even one with overloads, I get just a single message.  Odd.

All that said, the compiler has some special code to deal with vectorizeOnly() iterators, so (a) it could easily be that that's causing it in some way that I'm not picking up on, and (b) since @bmcdonald3 wanted me to update the message anyway... I could have the compiler generate the deprecation warning which only results in a single warning per loop rather than 3.